src/atoms/SvgIcon/dorris/Svg.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTheme } from 'emotion-theming';
+
+import { getColor } from '../../../utils/get-color';
+
+const Svg = ({ color, ...rest }) => {
```

Comment:
They're actually a bit different. By moving svg attributes that are common for all icons in a set, we get out of having to add them further down the tree. By having these attrs on the dorris Svg, we get out of having to add them on every single dorris icon. But they wouldn't necessarily apply to icons that are not in the set.

Reply:
About the colors, an alternative could be to let the default color be 'type'. Maybe that's better. I chose splashBackground because that fitted sol better, and the icon set was originally designed for sol. I'll commit the splashColor now and ask @Liselottehauerkind if I should change it after release.